lib/galaxy/tool_util/deps/container_resolvers/mulled.py

```diff
@@ -488,7 +514,7 @@ def __init__(self, app_info=None, namespace="local", hash_func="v2", auto_instal
         self.auto_init = self._get_config_option("involucro_auto_init", True)
 
     def resolve(self, enabled_container_types, tool_info, install=False, **kwds):
-        if tool_info.requires_galaxy_python_environment or self.container_type not in enabled_container_types:
+        if not self.docker_cli_available or tool_info.requires_galaxy_python_environment or self.container_type not in enabled_container_types:
```

Comment:
It would seem like it would be better to remove this from the list of default resolvers loaded if docker cli is unavailable - rather than loading it and just disabling it altogether? That way if it is explicitly loaded, it fails - like it should.

Reply:
Good idea, I'll do that.